Description: Adjust pathnames for FHS
Forwarded: not-needed
Author: Matej Vela <vela@debian.org>
Last-Update: 2011-05-13

Index: csh-20110313/pathnames.h
===================================================================
--- csh-20110313.orig/pathnames.h
+++ csh-20110313/pathnames.h
@@ -36,5 +36,5 @@
 #define	_PATH_DOTCSHRC		"/etc/csh.cshrc"
 #define	_PATH_DOTLOGIN		"/etc/csh.login"
 #define	_PATH_DOTLOGOUT		"/etc/csh.logout"
-#define	_PATH_LOGIN		"/usr/bin/login"
+#define	_PATH_LOGIN		"/bin/login"
 #define	_PATH_USRBIN		"/usr/bin"
